test_app/lib/screens/test_buttons/conference_service_test_buttons.dart

```diff
@@ -98,7 +107,12 @@ class ConferenceServiceTestButtons extends StatelessWidget {
     _dolbyioCommsSdkFlutterPlugin.conference
         .current()
         .then((conference) => _dolbyioCommsSdkFlutterPlugin.conference.leave())
-        .then((value) => Navigator.of(context).push(MaterialPageRoute(builder: (context) => const JoinConference(username: ''))))
+        .then((value) => Navigator.of(context)
+        .push(MaterialPageRoute(builder: (context) =>
```

Comment:
Pushing is not right here. We have to pop to the join screen. Please look at how the popup triggered by the red leave button works.

Reply:
Pushing was before my changes, but I can refactor it.